tests/cache/test-vary-cache.php

```diff
@@ -310,4 +321,69 @@ public function test__set_group_for_user_invalid( $group, $value, $expected_erro
 		$this->assertEquals( $expected_error_code, $actual_error_code, 'Incorrect error code' );
 	}
 
+	public function test__enable_encryption_true_invalid() {
+
+		$actual_result = Vary_Cache::enable_encryption( );
+
+		$this->assertWPError( $actual_result, 'Not WP_Error object' );
+		$actual_error_code = $actual_result->get_error_code();
+		$this->assertEquals( 'vary-cache-secrets-not-defined', $actual_error_code, 'Incorrect error code' );
+
+	}
+
+	public function test__enable_encryption_true_valid() {
```

Comment:
Will need to run this test in a separate process otherwise the `define` will persist across other tests.

See https://github.com/Automattic/vip-go-mu-plugins/blob/2280341588b2fe2fff26fec8b4cce984d2aa27df/tests/lib/utils/context.php#L40-L43

Reply:
Cool, was wondering on that